lib/hydra/pcdm.rb

```diff
@@ -24,8 +24,9 @@ def self.object? object
     end
 
     def self.file? file
-      return false unless file.respond_to? :type
-      file.type.include? RDFVocabularies::PCDMTerms.File
+      return false unless file.respond_to? :metadata_node
+      rdf_type = file.metadata_node.query(predicate: RDF.type, object: RDFVocabularies::PCDMTerms.File).map(&:object)
```

Comment:
can we do `file.metadata_node.type.include?( RDFVocabularies::PCDMTerms.File)`? The two lines are duplicating work because the query is only returning predicates that already match `PCDMTerm.File`